libheif/heif_version.h.in

```diff
@@ -31,6 +31,6 @@
 #define LIBHEIF_NUMERIC_VERSION @NUMERIC_VERSION@
 
 /* Version string */
-#define LIBHEIF_VERSION "@PACKAGE_VERSION@"
+#define LIBHEIF_VERSION "@PROJECT_VERSION@"
```

Comment:
Here I was not sure if this would conflict with the `configure.ac`. I am happy to revert this change if it is conflicting.

Reply:
Autoconf is using `PACKAGE_VERSION`, so please keep this.